### ISSUE TITLE
Creating unit tests for this plugin

### ISSUE THREAD

**Issue by pwlin** [pwlin/cordova-plugin-file-opener2#257]
I am trying to write some useful unit testing for this plugin which hopefully will help keeping the plugin function better.

Is that something that might be beneficial for the community? 

If so, is using [cordova-plugin-test-framework](https://github.com/apache/cordova-plugin-test-framework) the way to go?  Any feedback is really appreciated

**Comment by shnist** [comment]
yes that would be great! it's something that i looked into briefly and that plugin seemed like the way to go.
